index.html

```diff
@@ -0,0 +1,141 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+
+  <body>
+    <main id="main">
```

Comment:
You can't wrap your all body with the main element  because you can't wrap your header, and footer in the main tag instead of wrapping your whole website with the main you can wrap your changeable content